include/modules/clock.hpp

```diff
@@ -38,18 +38,19 @@ class Clock final : public ALabel {
     5 - tooltip-format
    */
   std::map<int, std::string const> fmtMap_;
-  uint cldMonCols_{3};                 // calendar count month columns
```

Comment:
The weird alignment is the result of clang-format, which is checked by the CI. Don't change the formatting or the CI will complain.

Reply:
Sorry about that. I ran `clang-format -i /include/modules/clock.hpp`. Hopefully this does the trick.